arch/CMakeLists.txt

```diff
@@ -1,4 +1,2 @@
-add_definitions(-D__ZEPHYR_SUPERVISOR__)
-
```

Comment:
Is this intentional?

Reply:
Sorry, I have pushed with your revert of #7567, it's an error of my part, I will push again without this patch